lib/Target/ARM/ARMPGLTOptimizer.cpp

```diff
@@ -126,6 +143,9 @@ void ARMPGLTOpt::replacePGLTUses(SmallVectorImpl<int> &CPEntries) {
         for (int i : CPEntries) {
           if (i == CPIndex) {
             const MachineConstantPoolEntry &Entry = ConstantPool->getConstants()[i];
+            // TODO(yln): should the cast below be a dynamic_cast?
+            // If not, then ACPC should probably defined here and there is not reason
+            // UsesToReplace.emplace_back needs to be done outside the loop.
```

Comment:
Agreed, the emplace_back can get moved up. This will always be a static cast, because we are in the ARM backend, and this is the ARM-specific constant pool value.